File: ConsoleId.cs
namespace Quark;

public static class ConsoleId
{
    private static readonly string[] Adjectives =
    [
        "Red", "Blue", "Gold", "Swift", "Brave", "Calm", "Dark", "Jade",
        "Iron", "Keen", "Lime", "Neon", "Opal", "Pink", "Rosy", "Sage",
        "Teal", "Volt", "Warm", "Zest", "Bold", "Cool", "Dusk", "Epic",
        "Fast", "Glow", "Hazy", "Icy", "Just", "Kiwi"
    ];

    private static readonly string[] Nouns =
    [
        "Switch", "Nova", "Pixel", "Spark", "Drift", "Flame", "Ghost",
        "Haven", "Ivory", "Jewel", "Karma", "Lunar", "Mango", "Nexus",
        "Orbit", "Prism", "Quest", "Radar", "Solar", "Titan", "Unity",
        "Vapor", "Wave", "Xenon", "Yield", "Zenith", "Apex", "Blaze",
        "Comet", "Delta"
    ];

    private static readonly Random _rng = new();

    public static string Generate()
    {
        string adj  = Adjectives[_rng.Next(Adjectives.Length)];
        string noun = Nouns[_rng.Next(Nouns.Length)];
        return $"{adj}-{noun}";
    }
}

File: android/app/src/main/java/io/github/quark/service/ConsoleId.kt
package io.github.quark.service

import kotlin.random.Random

object ConsoleId {

    private val adjectives = listOf(
        "Red", "Blue", "Gold", "Swift", "Brave", "Calm", "Dark", "Jade",
        "Iron", "Keen", "Lime", "Neon", "Opal", "Pink", "Rosy", "Sage",
        "Teal", "Volt", "Warm", "Zest", "Bold", "Cool", "Dusk", "Epic",
        "Fast", "Glow", "Hazy", "Icy", "Just", "Kiwi",
    )

    private val nouns = listOf(
        "Switch", "Nova", "Pixel", "Spark", "Drift", "Flame", "Ghost",
        "Haven", "Ivory", "Jewel", "Karma", "Lunar", "Mango", "Nexus",
        "Orbit", "Prism", "Quest", "Radar", "Solar", "Titan", "Unity",
        "Vapor", "Wave", "Xenon", "Yield", "Zenith", "Apex", "Blaze",
        "Comet", "Delta",
    )

    fun generate(): String {
        val adjective = adjectives[Random.nextInt(adjectives.size)]
        val noun = nouns[Random.nextInt(nouns.size)]
        return "$adjective-$noun"
    }
}
